packages/eslint-plugin-lit-a11y/package.json

```diff
@@ -31,6 +31,7 @@
     "dom5": "^3.0.1",
     "emoji-regex": "^9.0.0",
     "eslint": "^7.6.0",
+    "eslint-rule-extender": "^0.0.1",
```

Comment:
should we pin the version in case they do a breaking change since its <1?

Reply:
Sure